### ISSUE TITLE
TypeError: write() during report generation

### ISSUE THREAD

**Issue by rjsicko** [vladsaveliev/TargQC#3]
I'm getting the following error when running TargQC:
````
targqc /media/sf_BigShare/SCID_raw/Illumina/170607_M70314_0008_000000000-B5JJW-SCID-V1/Data/Intensities/BaseCalls/Alignment/101_S1.bam --bed /media/sf_BigShare/SCID/reference/171026-SCID_TSCA_from_manifest.sorted.annotated.bed -g hg19 -o test -t 4
bl8vbox
rsicko

/home/rsicko/miniconda2/bin/targqc /media/sf_BigShare/SCID_raw/Illumina/170607_M70314_0008_000000000-B5JJW-SCID-V1/Data/Intensities/BaseCalls/Alignment/101_S1.bam --bed /media/sf_BigShare/SCID/reference/171026-SCID_TSCA_from_manifest.sorted.annotated.bed -g hg19 -o test -t 4

----------------------------------------------------------------------
Using work directory /media/sf_BigShare/SCID/Illumina/Val-Run1/analysis/coverage/targqc/test/test/work
Found 1 BAM file

******************************************************
TargQC, target coverage evaluation tool. Version 1.4.4
******************************************************


_*lots of normal output omitted here*_

Saving reports...
Traceback (most recent call last):
  File "/home/rsicko/miniconda2/bin/targqc", line 329, in <module>
    main()
  File "/home/rsicko/miniconda2/bin/targqc", line 256, in main
    reannotate=reannotate)
  File "/home/rsicko/miniconda2/lib/python2.7/site-packages/targqc/__init__.py", line 101, in start_targqc
    is_debug=logger.is_debug, reannotate=reannotate, fai_fpath=fai_fpath)
  File "/home/rsicko/miniconda2/lib/python2.7/site-packages/targqc/general_report.py", line 374, in make_general_reports
    reannotate=reannotate)
  File "/home/rsicko/miniconda2/lib/python2.7/site-packages/targqc/general_report.py", line 574, in _build_report
    report.save_html(sample.targqc_html_fpath, caption='Target coverage statistics for ' + sample.name)
  File "/home/rsicko/miniconda2/lib/python2.7/site-packages/ngs_utils/reporting/reporting.py", line 503, in save_html
    tmpl_fpath=tmpl_fpath, data_dict=data_dict)
  File "/home/rsicko/miniconda2/lib/python2.7/site-packages/ngs_utils/reporting/reporting.py", line 393, in save_html
    tmpl_fpath=tmpl_fpath, data_dict=data_dict)
  File "/home/rsicko/miniconda2/lib/python2.7/site-packages/ngs_utils/reporting/reporting.py", line 1583, in write_html_report
    out_f.write(l)
TypeError: write() argument 1 must be unicode, not str
````

**Comment by vladsaveliev** [comment]
Hi rsicko,

Thanks for reporting the issue. For some reason the installer is picking a newer `ngs_utils` package version which appears to be incompatible with `targqc` published in bioconda: it errors with is a unicode-related bug occurring only under python 2, whereas `ngs_utils` was moved to python 3. I switched to python 3 in my personal projects so didn't pay much attention to back-compatibility with python 2 and didn't notice that. So I fixed that now and pushed a fixed version into my conda channel `vladsaveliev`. If you try and reinstall it by running:
```
conda remove -y targqc
conda install -y -c vladsaveliev -c bioconda targqc
```
It should hopefully run smoothly. Then I'll submit a request to update it in `bioconda` too.

Thanks again for debugging!
